lib/ship.rb

```diff
@@ -16,10 +16,6 @@ def hit
   end
 
   def sunk?
-    if @health == 0
-      true
-    else
-      false
-    end
+    @health == 0
```

Comment:
I found out from my independent challenge feedback this week that if you pass a conditional to a method it returns the boolean value so we don't need to write if or else or anything!